app/scripts/migrations/120.2.ts

```diff
@@ -175,7 +176,7 @@ function removeObsoletePhishingControllerState(
   if (!hasProperty(state, 'PhishingController')) {
     return;
   } else if (!isObject(state.PhishingController)) {
-    global.sentry.captureException(
```

Comment:
I left the majority of the migrations alone, because most used `global?.sentry?.captureException?.` (with optional chaining). I don't prefer the optional chaining, but, at least functionally it's doing as intended reliably. And this PR just got too large if I included those.

These two `global.sentry.captureException` references were easy to migrate to the new function.